backend/routes/api.php

```diff
@@ -23,4 +23,10 @@
         return UserResource::make($request->user());
     });
     Route::apiResource('users', UserController::class)->only(['index']);
+
+    //Users custom routes
+    Route::prefix('users')->group(function () {
+        Route::put('info', [UserController::class, 'updateInfo']);
```

Comment:
use resourceful routing for users update,

add 'update' to `Route::apiResource('users', UserController::class)->only(['index', 'update']);`

Reply:
Created new controller for profile

Reply:
@jescabillas what do you mean new controller for profile?

user's profile can be handled in the user controller